src/utils.cpp

```diff
@@ -21,68 +21,70 @@
 #include "utils.h"
 #include "debug.h"
 
-#include <glib.h>
 #include <QStandardPaths>
 #include <QUrlQuery>
-#include <QUrlQuery>
-
+#include <glib.h>
 
-typedef QPair<QString,QString> StringPair;
+typedef QPair<QString, QString> StringPair;
 
 namespace SystemSettings {
 
 void parsePluginOptions(const QStringList &arguments, QString &defaultPlugin,
-                        QVariantMap &pluginOptions)
-{
-    for (int i = 1; i < arguments.count(); i++) {
-        const QString &argument = arguments.at(i);
-        if (argument.startsWith("settings://")) {
-            QUrl urlArgument(Utilities::getDestinationUrl(argument));
-            /* Find out which plugin is required. If the first component of the
-             * path is "system", just skip it. */
-            QStringList pathComponents =
-                urlArgument.path().split('/', QString::SkipEmptyParts);
-            int pluginIndex = 0;
-            if (pathComponents.value(pluginIndex, "") == "system")
-                pluginIndex++;
-            defaultPlugin = pathComponents.value(pluginIndex, QString());
-            /* Convert the query parameters into options for the plugin */
-            QUrlQuery query(urlArgument);
-            Q_FOREACH(const StringPair &pair, query.queryItems()) {
-                pluginOptions.insert(pair.first, pair.second);
-            }
-        } else if (!argument.startsWith('-')) {
-            defaultPlugin = argument;
-        } else if (argument == "--option" && i + 1 < arguments.count()) {
-            QStringList option = arguments.at(++i).split("=");
-            // If no value is given, insert an empty string
-            pluginOptions.insert(option.at(0), option.value(1, ""));
+                        QVariantMap &pluginOptions) {
```

Comment:
Please undo the formatting changes, or it's making it much harder to see what changed.